python/cudf/cudf/tests/test_cuda_array_interface.py

```diff
@@ -198,9 +200,9 @@ def test_cuda_array_interface_pytorch():
 
     index = cudf.core.index.GenericIndex([1, 2, 8, 6])
     tensor = torch.tensor(index)
-    got = cudf.Series(tensor)
+    got = cudf.core.index.GenericIndex(tensor)
```

Comment:
Should we be invoking the `Index()` factory here instead?

Reply:
Yup, good call. Changed it.